### PR TITLE
NFDiv-3789 conditional order app doc pack

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**
https://tools.hmcts.net/jira/browse/NFDIV-3789

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingConditionalOrderNotification.java [hunk 1]
```diff
@@ -32,16 +33,13 @@
...
 public class AwaitingConditionalOrderNotification implements ApplicantNotification {
 
-    @Autowired
-    private CommonContent commonContent;
-
-    @Autowired
-    private NotificationService notificationService;
-
-    @Autowired
-    private ApplyForConditionalOrderPrinter applyForConditionalOrderPrinter;
```
Comment (ed14537) [inline src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingConditionalOrderNotification.java:44]: Looks like this printer class isn't being used any more? Can it be deleted?

File: src/main/java/uk/gov/hmcts/divorce/systemupdate/service/task/GenerateApplyForConditionalOrderDocument.java
Comment (ed14537) [inline src/main/java/uk/gov/hmcts/divorce/systemupdate/service/task/GenerateApplyForConditionalOrderDocument.java:1]: I would rename this to ApplyForConditionalOrderTemplateContent for consistency

File: src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingConditionalOrderNotification.java (hunk 1 above)
Comment (BWeeks-JusticeGovUK) [reply src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingConditionalOrderNotification.java:44]: Yep - deleted

File: src/main/java/uk/gov/hmcts/divorce/systemupdate/service/task/GenerateApplyForConditionalOrderDocument.java
Comment (BWeeks-JusticeGovUK) [reply src/main/java/uk/gov/hmcts/divorce/systemupdate/service/task/GenerateApplyForConditionalOrderDocument.java:1]: Agreed - done

**Review by ed14537 — APPROVED** [review]

**Pull request merged**